libavfilter/dnn/dnn_backend_tf.h

```diff
@@ -29,11 +29,57 @@
 
 #include "../dnn_interface.h"
 
+/**
+ * @brief Load model from the model.
+ *
+ * It allocates and initializes the DNN model
+ * with the model parameters. If the graph cannot
+ * be read by the TensorFlow C API, the function
+ * tries loading it with native backend.
+ *
+ * @param model_filename name of the model file to load
+ * @param func_type function type of model
+ * @param options model execution options
+ * @param filter_ctx filter context
+ * @return The pointer to DNNModel instance of the loaded model
+ * @retval NULL if the model cannot be loaded.
+ * All allocated are freed by the function.
+ */
 DNNModel *ff_dnn_load_model_tf(const char *model_filename, DNNFunctionType func_type, const char *options, AVFilterContext *filter_ctx);
 
+/**
+ * @brief Execute TensorFlow model in synchronous mode.
+ *
+ * It parses the data from the input frame, preprocesses
+ * the input frame data if required and fills the input
+ * Tensor. Then it run the TensorFlow session and fills
+ * the output after post processing it if required.
+ *
+ * Currently only single output is supported.
```

Comment:
maybe we can pending this after Fu Ting's patch is merged, he added multiple outputs for tf backend.

Reply:
Sure. Do you mean - this function's documentation or the line "Currently only single output is supported"?

Reply:
yes